src/umpire/util/allocation_statistics.cpp

```diff
@@ -0,0 +1,40 @@
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
@jdahm - please create a JIRA ticket about finding a better home for this method.